libsignal/src/main/java/org/session/libsignal/crypto/MnemonicCodec.kt

```diff
@@ -94,10 +78,8 @@ class MnemonicCodec(private val loadFileContents: (String) -> String) {
         var result = ""
         val n = truncatedWordSet.size.toLong()
         // Check preconditions
-        if (words.size < 12) { throw DecodingError.InputTooShort
-        }
-        if (words.size % 3 == 0) { throw DecodingError.MissingLastWord
-        }
+        if (words.size < 12) throw DecodingError.InputTooShort
+        if (words.size % 3 == 0) throw DecodingError.MissingLastWord
```

Comment:
I think this got changed (or at least flagged) on the `on-1` PR - mod 3 is a strange way to determine a missing last word.

Reply:
IDK, it seems intentional to me. It's probably to keep consistent with other mnemonics you can use in Monero.